README.md

```diff
@@ -1,5 +1,17 @@
 # compas-scl-xsd
```

Comment:
(Nothing to do with README) I would also add a .gitignore file for things like .classpath, .gradle, .project, .settings, etc..